scripts/test/js_test.sh

```diff
@@ -12,6 +12,9 @@ then
     export CMD="node ./node_modules/mocha/bin/_mocha --watch"
 else
     export CMD="node ./node_modules/mocha/bin/_mocha"
+    if [[ ! -z "$BAIL" ]]; then
```

Comment:
After this PR is merged changes like these which may affect multiple projects should be ported to the cookiecutter: https://github.com/mitodl/cookiecutter-djangoapp/. Feel free to just make an issue in the cookiecutter linking to this PR, the porting doesn't have to happen right away

Reply:
👍